### ISSUE TITLE
Configuration of Bot Detection/ReCAPTCHA

### ISSUE THREAD

**Issue by Moggers** [auth0/terraform-provider-auth0#78]
### Describe the problem you'd like to have solved

Under `Security` -> `Attack Prevention` -> `Bot Detection` of the Auth0 management UI there is a section for enabling ReCAPTCHAv2 in the universal login. Currently there doesn't *seem* to be a way to configure this via the terraform module.

### Describe the ideal solution

Being able to enable ReCAPTCHA and specify keys via terraform.

### Alternatives and current workarounds

Currently a manually process.

**Comment by sergiughf** [comment]
Hey @Moggers, thanks for opening this issue. Unfortunately we're unable to add this feature yet due to the endpoint on the management API missing. It currently covers only the following: https://auth0.com/docs/api/management/v2#!/Attack_Protection/get_breached_password_detection. 

Will update this whenever this is available.

**Comment by zachfeld** [comment]
hi @sergiught is there any place I can track the addition of this endpoint within management API? It's something I'd like to be able to accomplish as well. Thanks!

**Comment by Aaronius** [comment]
This is the biggest gap we're facing currently. Not just configuring the captcha but also the risk assessment logs, IP allow list, etc.

**Comment by davedash** [comment]
Is this still an issue:

https://auth0.com/docs/api/management/v2#!/Attack_Protection/patch_suspicious_ip_throttling

**Comment by willvedd** [comment]
> Is this still an issue:
> 
> https://auth0.com/docs/api/management/v2#!/Attack_Protection/patch_suspicious_ip_throttling

@davedash Yes, suspicious IP throttling is not the same as bot detection. This feature is still blocked by a lack of an API endpoint. Rest assured, we're keen on delivering this feature as soon as we're able!

**Comment by davedash** [comment]
@willvedd Lol, I was so excited, and as I was editing my terraform config, I realized the same thing.  Thanks.

**Comment by sergiught** [comment]
Hey folks, unfortunately no updates yet on this. We are still waiting for the endpoints to get added to the Management API. As soon as that happens we'll immediately add support within our tooling. We'll be sure to post an update in this issue when that happens. Appreciate everyones patience! 🙏🏻

**Comment by EQnews** [comment]
Are there any further updates/timelines on this one?

**Comment by cbeardsmore** [comment]
Any updates on this one? Its been almost 2 years. Thats a decent lag time for a. Terraform provider.

**Comment by EQnews** [comment]
Hi - is there any update on this?  I was told that Terraform capability depends on the release of 'Bot Detection Public API' which is on the roadmap planned for 2024 Q1 (FEB - APR).  Is this on track?

**Comment by kxs-mdoyon** [comment]
Hello, Can we please get specific dates for this re;ease?

**Comment by igobl** [comment]
> our Auth0 customer rep told us this week that it's now scheduled for H2 2024 😬
Ran into this issue as well, it's a little annoying. Is anybody from Auth0 able to confirm this?

**Comment by deBFM** [comment]
_Bot Detection endpoints will be added to the Management API sometime between July and September of 2024. It is currently on the Auth0 roadmap product roadmap._

Source: [Bot Detection configuration via Management API](https://community.auth0.com/t/bot-detection-configuration-via-management-api/136328)

**Comment by dmunch** [comment]
Related: https://github.com/auth0/auth0-deploy-cli/issues/649
